src/app/layout.tsx

```diff
@@ -1,21 +1,22 @@
-import './globals.css'
-import { Inter } from 'next/font/google'
+import "./globals.css";
+import { Inter } from "next/font/google";
 
-const inter = Inter({ subsets: ['latin'] })
+const inter = Inter({ subsets: ["latin"] });
 
 export const metadata = {
-  title: 'Create Next App',
-  description: 'Generated by create next app',
-}
+  title: "Faker: JSON Placeholder",
+  description:
```

Comment:
I don't think this is being shown right now?

Reply:
hmm i'm not sure about that because it is working as expected in mine 
<img width="415" alt="Screenshot 2023-06-03 at 17 58 19" src="https://github.com/debbyalmadea/faker/assets/88815144/11f9d41e-7d85-4025-baea-0cfbc622227d">